tools/shell/linenoise.cpp

```diff
@@ -570,6 +570,9 @@ uint32_t linenoiseComputeRenderWidth(const char* buf, size_t len) {
 }
 
 int getBufCharPos(struct linenoiseState* l) {
+    if (l->chars == 0) {
```

Comment:
I don't really understand what this function is doing? Can you add some comments here?

Reply:
This function will most likely not be required, I debugged this function and all it is doing is returning the proper position of the cursor from left.

But `l->pos` already holds this, that's why in DuckDB's codebase everywhere only `l->pos` has been used instead of calling a separate function.

I'm currently fixing this issue.